dags/ltv.py

```diff
@@ -0,0 +1,89 @@
+import json
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.subdag_operator import SubDagOperator
+from utils.dataproc import (
+    moz_dataproc_pyspark_runner,
+    copy_artifacts_dev,
+    get_dataproc_parameters,
+)
+
+EXPORT_TO_AVRO = True
+
+default_args = {
+    "owner": "amiyaguchi@mozilla.com",
+    "depends_on_past": True,
+    "start_date": datetime(2020, 3, 15),
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "amiyaguchi@mozilla.com",
+        "bmiroglio@mozilla.com",
+    ],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 3,
+    "retry_delay": timedelta(minutes=30),
+}
+
+dag = DAG("ltv_daily", default_args=default_args, schedule_interval="@daily")
```

Comment:
@benmiroglio Are there any dependencies for this job that need to be taken into consideration?

Reply:
the only dependent dataset is`search.search_rfm`.

Reply:
Do you know if that table is scheduled in airflow, so it can be set as a dependency? A quick cursory glance at the `main_summary` dag doesn't turn up anything.